charts/timed/templates/ingress.yaml

```
@@ -17,25 +17,35 @@ spec:
         paths:
           - path: /admin/
             backend:
-              serviceName: {{ $.Values.backend.service.name }}
-              servicePort: {{ $.Values.backend.service.externalPort }}
+              service:
+                name: {{ $.Values.backend.service.name }}
+                port:
+                  number: {{ $.Values.backend.service.externalPort }}
           - path: /static/
             backend:
-              serviceName: {{ $.Values.backend.service.name }}
-              servicePort: {{ $.Values.backend.service.externalPort }}
+              service:
+                name: {{ $.Values.backend.service.name }}
+                port:
+                  number: {{ $.Values.backend.service.externalPort }}
           - path: /api/
             backend:
-              serviceName: {{ $.Values.backend.service.name }}
-              servicePort: {{ $.Values.backend.service.externalPort }}
+              service:
+                name: {{ $.Values.backend.service.name }}
+                port:
+                  number: {{ $.Values.backend.service.externalPort }}
```

I'd recommend to change to named ports instead of using the port number:
```suggestion
                  name: {{ $.Values.backend.service.name }}
```

---

changed to named ports